pkg/swarm/swarm.go

```diff
@@ -61,6 +61,11 @@ func (a Address) Bytes() []byte {
 	return a.b
 }
 
+// ByteString returns raw Address string without decoding.
```

Comment:
Should be `encoding` instead of `decoding` in the comment, as tis function encodes?

Reply:
Yes, sorry 🙈

Reply:
Introduced a typo to fix a typo :trollface: 😂